machine/wnc/wnc_model/busybox/conf/config

```diff
@@ -0,0 +1,6 @@
+CONFIG_SYS_EEPROM=y
+CONFIG_SYS_EEPROM_DEVICE_MTD=y
+CONFIG_SYS_EEPROM_MTD_DEVICE="/dev/sda3"
```

Comment:
Yes, I create /dev/sda3 as my TLV EEPROM, and partition size is 2K byte.
Below is my partition info via command "cat /proc/partitions":
*******************************
major minor  #blocks  name

   8        0  250059096 sda
   8        1       2048 sda1
   8        2     131072 sda2
   8        3          2 sda3
*******************************
If NOS has been installed, it will add /dev/sda4 at partition table.

And below is my disk info via command "sgdisk -p /dev/sda":
*******************************
Disk /dev/sda: 500118192 sectors, 238.5 GiB
Logical sector size: 512 bytes
Disk identifier (GUID): 3DDAF4F9-9F57-4D63-84EC-F61A06A2F0BA
Partition table holds up to 128 entries
First usable sector is 34, last usable sector is 500118158
Partitions will be aligned on 2048-sector boundaries
Total free space is 499851881 sectors (238.3 GiB)

Number  Start (sector)    End (sector)  Size       Code  Name
   1            2048            6143   2.0 MiB     EF02  GRUB-BOOT
   2            6144          268287   128.0 MiB   3000  ONIE-BOOT
   3          268288          268291   2.0 KiB     8301  ONIE-EEPROM
*******************************
Will it be a problem if I use /dev/sda3 as TLV EEPROM?
I will pull a request to show how /dev/sda3 created via install-arch.